launch/tier4_planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py

```diff
@@ -172,25 +172,75 @@ def launch_setup(context, *args, **kwargs):
         extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
     )
 
+    # obstacle velocity planner
+    obstacle_velocity_planner_param_path = os.path.join(
+        get_package_share_directory("tier4_planning_launch"),
+        "config",
+        "scenario_planning",
+        "lane_driving",
+        "motion_planning",
+        "obstacle_velocity_planner",
+        "obstacle_velocity_planner.param.yaml",
+    )
+    with open(obstacle_velocity_planner_param_path, "r") as f:
+        obstacle_velocity_planner_param = yaml.safe_load(f)["/**"]["ros__parameters"]
+    obstacle_velocity_planner_component = ComposableNode(
+        package="obstacle_velocity_planner",
+        plugin="motion_planning::ObstacleVelocityPlannerNode",
+        name="obstacle_velocity_planner",
+        namespace="",
+        remappings=[
+            ("~/input/trajectory", "obstacle_avoidance_planner/trajectory"),
+            ("~/input/odometry", "/localization/kinematic_state"),
+            ("~/input/objects", "/perception/object_recognition/objects"),
+            ("~/output/trajectory", "/planning/scenario_planning/lane_driving/trajectory"),
+            ("~/output/velocity_limit", "/planning/scenario_planning/max_velocity_candidates"),
+            ("~/output/clear_velocity_limit", "/planning/scenario_planning/clear_velocity_limit"),
+            ("~/output/stop_reasons", "/planning/scenario_planning/status/stop_reasons"),
+        ],
+        parameters=[
+            common_param,
+            obstacle_velocity_planner_param,
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
     container = ComposableNodeContainer(
         name="motion_planning_container",
         namespace="",
         package="rclcpp_components",
         executable=LaunchConfiguration("container_executable"),
         composable_node_descriptions=[
             obstacle_avoidance_planner_component,
-            obstacle_stop_planner_component,
         ],
     )
 
+    obstacle_stop_planner_loader = LoadComposableNodes(
+        composable_node_descriptions=[obstacle_stop_planner_component],
+        target_container=container,
+        condition=IfCondition(LaunchConfiguration("use_obstacle_stop_planner")),
+    )
+
+    obstacle_velocity_planner_loader = LoadComposableNodes(
+        composable_node_descriptions=[obstacle_velocity_planner_component],
+        target_container=container,
+        condition=IfCondition(LaunchConfiguration("use_obstacle_velocity_planner")),
```

Comment:
[imo]
I think it is better to use `LaunchConfigurationEquals` like control.launch.py, because you can explicitly indicate that only one of them (stop_planner and velocity_planner) will be launched.

https://github.com/tier4/autoware_launch/blob/964f4f0b751923815c0bde043a7820246ff55b5c/control_launch/launch/control.launch.py#L259-L268

Reply:
Thank you for letting me know. I'll modify it like this at last.
https://github.com/tier4/autoware_launch/pull/330/files